pyrate/core/gamma.py

```diff
@@ -249,7 +443,11 @@ def gamma_header(ifg_file_path, params):
     """
     dem_hdr_path = params[cf.DEM_HEADER_FILE]
     header_paths = get_header_paths(ifg_file_path, params[cf.HDR_FILE_LIST])
-    combined_headers = manage_headers(dem_hdr_path, header_paths)
+    if len(header_paths) == 2:
+        baseline_path = cf.baseline_paths_for(ifg_file_path, params)
+    else:
+        baseline_path = None  # don't read baseline files for DEM
+    combined_headers = manage_headers(dem_hdr_path, header_paths, baseline_path)
```

Comment:
Just an observation, that I think this function call makes baseline files a mandatory input for a GAMMA-based PyRate run...
baseline_path could be made an optional arg to `manage_headers`, defaulting to 'None' ...?

Reply:
see https://github.com/GeoscienceAustralia/PyRate/pull/302/commits/fa2a65850f4aec5bfc26712c9339f19b6edc51ee